### PR TITLE
AirfRANS: Lower LR (3e-4) + T_max=10 at 4L/256d golden config

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The pressure-weight breakthrough (0.00435) used lr=3e-4 + T_max=10 at 3L/192d. Even WITHOUT pressure weighting, these hyperparameters might be better than the current golden config (lr=7e-4, T_max=5). We isolate the hyperparameter effect: if lr=3e-4+T_max=10 beats lr=7e-4+T_max=5 at 4L/256d WITHOUT pressure weighting, then the improvement from pressure weighting would compound on top of already-better hyperparams.

## Instructions
```bash
cd target/icml2026 && SENPAI_MAX_EPOCHS=9999 SENPAI_TIMEOUT_MINUTES=180 python train.py \
  --dataset airfrans \
  --airfrans_task full \
  --optimizer adamw \
  --lr 3e-4 \
  --cosine-t-max 10 \
  --grad-clip 1.0 \
  --weight-decay 1e-2 \
  --no-use-ema \
  --enable-fourier \
  --model-layers 4 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --epochs 999 \
  --seed 789 \
  --agent luffy \
  --wandb-name "luffy/airfrans-4L256d-lr3e4-tmax10-golden" \
  --wandb-group "airfrans-lr-tmax-sweep"
```

Note: Using seed=789 (best from multi-seed characterization).

Report val_primary/surface_mse, best epoch, total epochs.

## Baseline
Current best: **0.007264** (4L/256d, lr=7e-4, T_max=5, golden config)
External target: 0.0043

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2807 because ADVISOR: Closing — 4L/256d architecture is fully superseded by 3L/256d (0.001479 vs your 4L baseline). lr=3e-4 was also confirmed suboptimal. You'll be reassigned to a current experiment..
